src/maintenance/performance.rst

```diff
@@ -268,7 +270,7 @@ that will occur giving you a quick response and when views will be updated
 which will take a long time. (A 10 million document database took about 10
 minutes to load into CouchDB but about 4 hours to do view generation).
 
-In a cluster, "stale" requests are serviced by a fixed set of shards in order
+In a cluster, "stable" requests are serviced by a fixed set of shards in order
```

Comment:
Hm. This is a good change (not talking about `stale`), but down in line 279 it talks about `stale=ok` and `stale=update_after` again.

Similarly up in line 268 we talk about "stale" again.

If you want to fully expunge the concept of stale requests, we should do it in this entire section. I'd be +1 on that work but don't have time to help today.

Otherwise, I'd reverse the change and we can do that at a later date.

Reply:
> Hm. This is a good change (not talking about `stale`), but down in line 279 it talks about `stale=ok` and `stale=update_after` again.

Those lines refer to a deprecated parameter, while 0526b9b was fixing a misleading typo (the meaning by this is exactly the opposite of the behavior).

> 
> Similarly up in line 268 we talk about "stale" again.
> 
> If you want to fully expunge the concept of stale requests, we should do it in this entire section. I'd be +1 on that work but don't have time to help today.
> 
> Otherwise, I'd reverse the change and we can do that at a later date.

I only wanted to "leave the campground cleaner than I found it". To avoid scope creep in this PR, I've pushed 1fce16e to revert the fix for the misleading typo ( 0526b9b ).